src/app/components/client/ProgressCard.tsx

```diff
@@ -84,20 +84,34 @@ export const ProgressCard = (props: Props) => {
         </button>
       </div>
       <div className={styles.progressStatsWrapper}>
+        
+        {/* Manually fixed */}
         <div className={styles.progressItem}>
           <div className={styles.progressStat}>
             <Image src={ExploringLaptopPlus} alt="" width="50" height="50" />
             <span>{props.resolvedByYou}</span>
           </div>
-          <p>{l10n.getString("progress-card-resolved-by-you-headline")}</p>
+          <p>{l10n.getString("progress-card-manually-fixed-headline")}</p>
         </div>
-        <div className={styles.progressItem}>
+
+        {/* Auto-removed */}
+        <div
+          className={`${styles.progressItem} ${
+            !props.isPremiumUser && styles.greyedOut
+          }`}
+        >
           <div className={styles.progressStat}>
             <Image src={ExploringLaptopMinus} alt="" width="50" height="50" />
             <span>{props.autoRemoved}</span>
           </div>
-          <p>{l10n.getString("progress-card-auto-removed-headline")}</p>
+          <p>
+            {!props.isPremiumUser && <LockIcon alt="" width="10" height="10" />}
```

Comment:
Since this lock icon does communicate something to the user, maybe it should have a (localised) alt text? Perhaps something like `(only available with Monitor Premium)`?